src/sources/close_crm/transform.js

```diff
@@ -48,8 +49,8 @@ function processEvent(inputEvent) {
   return message;
 }
 
-function process(inputEvent) {
-  const { event } = inputEvent;
+function process(payload) {
+  const event = getBodyFromV2SpecPayload(payload);
```

Comment:
```suggestion
  const { event } = getBodyFromV2SpecPayload(payload);
```
test case contract got updated, so this is passing through

Reply:
As shown in the previous comment, we are destructing the `event` from v1 spec previously and now, we are taking the `json parsed request.body` from the `v2` spec instead. Both the cases still have the payload's `event` still available.